be/src/http/action/stream_load.cpp

```diff
@@ -527,10 +527,12 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req, StreamLoadContext*
             return Status::InvalidArgument("Invalid load_dop format");
         }
     }
+    int32_t rpc_timeout_ms = config::txn_commit_rpc_timeout_ms;
```

Comment:
We need to make sure timeout positive

Reply:
We need a config framework to check the validity of the configuration instead of checking it every time it is used

Reply:
agree!

Reply:
`txn_commit_rpc_timeout_ms` is 10 seconds in 2.3, do we need to adjust it?  @meegoo @nshangyiming